src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetDeclaredSymbolAPITests.cs

```diff
@@ -5267,6 +5267,380 @@ void F(int x = 2, = 3) { }
             Assert.Same(symbol1.ContainingSymbol, symbol2.ContainingSymbol);
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/74348")]
+        public void ObjectInitializerIncompletePropertyValueDeclaration01()
```

Comment:
>Declaration [](http://example.com/codeflow?start=60&length=11)

Regarding the "Declaration" word in the name of the test. What is it supposed to convey?

Reply:
It should be "Assignment" I think, I'll change that in the next commits

Reply:
Renamed those methods